packages/taco/src/conditions/context/context.ts

```diff
@@ -55,6 +58,10 @@ const EXPECTED_AUTH_PROVIDER_TYPES: Record<string, AuthProviderType[]> = {
   ],
 };
 
+export const AUTOMATICALLY_INJECTED_CONTEXT_PARAMS = [
+  // These context parameters are automatically injected on the node side.
+  SIGNING_CONDITION_OBJECT_CONTEXT_VAR,
+];
 export const RESERVED_CONTEXT_PARAMS = [USER_ADDRESS_PARAM_DEFAULT];
```

Comment:
```suggestion
export const RESERVED_CONTEXT_PARAMS = [USER_ADDRESS_PARAM_DEFAULT, SIGNING_CONDITION_OBJECT_CONTEXT_VAR];
```

so that users can't use this context variable

Reply:
✔️  - it's already prevented via `AUTOMATICALLY_INJECTED_CONTEXT_PARAM`, but your point is well taken that it is also a reserved context param - probably good to add anyway for code maintenance